html/elements/input/range.json

```diff
@@ -36,7 +36,7 @@
                 "version_added": "3.1"
               },
               "safari_ios": {
-                "version_added": "5.1"
+                "version_added": "5"
```

Comment:
Discussed in https://github.com/mdn/browser-compat-data/pull/11175#discussion_r671003559